src/ess/reduce/nexus.py

```diff
@@ -71,6 +71,7 @@
 
 def load_detector(
     file_path: Union[FilePath, NeXusFile, NeXusGroup],
+    data_slice=(),
```

Comment:
Not happy with the name, but I don't have a better idea right now. @nvaytet @jl-wynen do you?

Reply:
some alternative names for the argument, kind of in the order I like them most:

* `selection`
* `sel`
* `slice_`
* `nxslice`
* `load_slice`
* `slice_index`
* `index`
* `part`